router/h2/src/test/scala/io/buoyant/router/h2/H2AddForwardedHeaderTest.scala

```diff
@@ -0,0 +1,113 @@
+package io.buoyant.router.h2
+
+import java.net.{InetAddress, InetSocketAddress}
+import com.twitter.finagle._
+import com.twitter.finagle.buoyant.h2.{Method, Request, Response, Stream}
+import com.twitter.util.{Future, Time}
+import io.buoyant.router.http.ForwardedHeaderLabeler
+import io.buoyant.test.FunSuite
+
+class H2AddForwardedHeaderTest extends FunSuite {
+  val OkSvc = Service.mk[Request, Response] { req =>
+
+    val rsp = Response(req.headers, Stream.const("Hello, World!"))
```

Comment:
Maybe use Stream.empty so that you don't need to worry about releasing the data frames